README.md

```diff
@@ -35,7 +36,12 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+instructor.findOne.({name: "Andy"}),
+dunction(err, andy){
+  andy.wishlist_items.push({description: "Resin laying"});
+  andy.save();
+};
+
 ```
```

Comment:
The callback needs to be passed in as a second argument to `findOne`.

You have a number of syntax errors here as well.